src/renderer/components/+workloads-pods/pod-container-env.tsx

```diff
@@ -124,75 +149,35 @@ export const ContainerEnvironment = observer((props: ContainerEnvironmentProps)
 
     if (!secret) return null;
 
-    return Object.keys(secret.data).map(key => {
-      const secretKeyRef = {
-        name: secret.getName(),
-        key,
-      };
-
-      const value = (
-        <SecretKey
-          reference={secretKeyRef}
-          namespace={namespace}
-        />
-      );
-
-      return (
-        <div className="variable" key={key}>
-          <span className="var-name">{key}</span>
-          :
-          {value}
+    return Object.keys(secret.data)
+      .map(name => (
+        <div
+          className="variable"
+          key={name}
+          data-testid={`envFrom-secret-${secretName}-${name}`}
+        >
+          <span className="var-name">{name}</span>
+          {`= `}
+          <SecretKey
+            secret={secret}
+            field={name}
+          />
```

Comment:
This part seems to be duplicated quite many times. Maybe you could introduce non-exported component to bottom of file which does the trick? :)